### DISCUSSION TITLE
Is there any way to view the real data of the object's data parts and parity parts

### DISCUSSION THREAD

**Discussion by MoonJustry** [minio/minio#14023] [Q&A] [answered]
I want to debug relevant information through the specific data block and parity block of the object, including the meta information and the real content of the object (which can not be obtained through the server API at this time)。
For example: 
I collect the object xl.meta through `mc admin inspect myminio/.minio.sys/config/iam/users/SECRETID/identity.json/xl.meta`, which is a small object, the data is inline in the meta information.
I can view the meta information through the command `xl-meta inspect.xxxx.zip`, but cannot view the real total content of the object.

**Comment by klauspost** [comment, marked as answer]
Use `mc admin inspect myminio/.minio.sys/config/iam/users/SECRETID/identity.json/**` to grab all files.

Docs: https://github.com/minio/minio/tree/master/docs/debugging#remotely-inspecting-backend-data

**Reply by MoonJustry** [reply]
Thank you for your reply.
I execute this cmd `mc admin inspect myminio/.minio.sys/config/iam/users/SECRETID/identity.json/**`, 'inspect.7623675c.zip' downloaded. But I don't know how to get the real content of this object from the zip file.

**Reply by klauspost** [reply]
@MoonJustry If you want that, you need to reassemble data. That is not trivial. I can only guide you to look through the source code.

This is probably the best place to start: https://github.com/minio/minio/blob/f527c708f248b7fc9731db51fa4d815b5ae00e23/cmd/erasure-object.go#L247

**Reply by MoonJustry** [reply]
I thought there would be some debugging tools like xl-meta.
Thank you very much.
